build.gradle

```diff
@@ -211,6 +211,11 @@ project('segmentstore:storage:impl') {
         testCompile project(':test:testcommon')
         testCompile project(path:':segmentstore:storage', configuration:'testRuntime')
     }
+    test {
```

Comment:
The changes in the build file do not seem to be related to the issue.

Reply:
Yeah these are temporary changes , needed for building the image as there are many sporadic unit test failures. Will revert it once the changes are confirmed.